.github/workflows/CI.yml

```diff
@@ -573,6 +574,46 @@ jobs:
           #    docker tag $IMAGE_NAME $REPO_NAME:$TAG
           #    docker push $REPO_NAME:$TAG
 
+  e2etest-public-ecr-release:
+    runs-on: ubuntu-latest
+    needs: [e2etest-preparation]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache if success
+        id:  e2etest-public-ecr-release
+        uses: actions/cache@v2
+        with:
+          path: |
+            VERSION
+          key: e2etest-public-ecr-release-${{ github.run_id }}
+
+      - name: Configure AWS Credentials
+        if: steps.e2etest-public-ecr-release.outputs.cache-hit != 'true'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
+          aws-region: us-east-1
+
+      - name: restore cached images
+        if: steps.e2etest-public-ecr-release.outputs.cache-hit != 'true'
+        uses: actions/cache@v2
+        with:
+          path: build/packages
+          key: "cached_packages_${{ github.run_id }}"
+
+      - name: upload to public ECR
+        if: steps.e2etest-public-ecr-release.outputs.cache-hit != 'true'
+        run: |
+          aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/aws-otel-test
```

Comment:
Think it's better to use the official docker actions for login and build/push

https://github.com/docker/build-push-action

Reply:
aws-actions/amazon-ecr-login@v1 is for private ecr. 
docker/login-action@v1  is for dockerhub. 
We are not building the image just loading

Reply:
I added the github action for login but it seems there is no action to only push